lib/App/DuckPAN/Web.pm

```diff
@@ -539,6 +541,19 @@ sub request {
 	return $response;
 }
 
+
+sub run_on_ready_script {
+	my ($str) = @_;
+
+	# any unescaped script tags at this stage will XSS the page - so strip them
+	$str =~ s|</?script>||gi;
```

Comment:
@moollaza There could be other stuff in the `script` tag - we should use something like:

```perl
my $dom = Mojo::DOM->new->parse($str);
$dom->find('script')->map('strip');
$str = $dom->to_string;
```